src/parser/creative_companion_parser.js

```diff
@@ -41,10 +41,12 @@ export function parseCreativeCompanion(creativeElement, creativeAttributes) {
         .childrenByName(companionResource, 'StaticResource')
         .reduce((urls, resource) => {
           const url = parserUtils.parseNodeText(resource);
+          const creativeType = resource.getAttribute('creativeType') || null;
```

Comment:
there was no need for this change, now `creativeType` constant is created on every loop of reduce